### ISSUE TITLE
Ternary statements in ember/order-in-components

### ISSUE THREAD

**Issue by craigbilner** [netguru/eslint-plugin-ember#25]
Should ternary statements be classified as default properties? Currently they're considered as an unknown property.

**Comment by michalsnik** [comment]
That's a good question @craigbilner. Let me come back to you after short discussion with my team mates.

Btw. what is your opinion?

**Comment by craigbilner** [comment]
@michalsnik, I think they should:

```js
foo: Ember.Testing ? 1 : 2
```

is a default value of either 1 or 2 to me

**Comment by michalsnik** [comment]
Okay, we had a brief talk and everyone has the same thoughts @craigbilner :) This should be treated as default property indeed 👍

**Comment by michalsnik** [comment]
https://github.com/netguru/eslint-plugin-ember/releases/tag/v2.2.2
